scalanet/src/io/iohk/scalanet/peergroup/kademlia/Xor.scala

```diff
@@ -0,0 +1,11 @@
+package io.iohk.scalanet.peergroup.kademlia
+
+import scodec.bits.BitVector
+
+object Xor {
+
+  def d(a: BitVector, b: BitVector): BigInt = {
+    assert(a.length == b.length)
+    BigInt((a xor b).toBin, 2)
```

Comment:
Isn't there a better conversion way than rendering 160-character string? e.g. `BigInt(1, (a xor b).toByteArray)`?

Reply:
There is!